kad-dht/README.md

```diff
@@ -268,23 +268,55 @@ heuristic of the value to make the decision.
 
 ### Content provider advertisement and discovery
 
-Nodes must keep track of which nodes advertise that they provide a given key
-(CID). These provider advertisements should expire, by default, after 24 hours.
-These records are managed through the `ADD_PROVIDER` and `GET_PROVIDERS`
+There are two things at play with regard to provider record (and therefore content)
+liveness and reachability:
+
+Content providers need to make sure that their content is reachable, despite peer churn
+and nodes that store and serve provider records need to make sure that the Multihashes whose 
+records they store are still served by the content provider.
```

Comment:
How do they do this? Is this happening today on the IPFS DHT?

Reply:
I think the above phrasing implies this being an active process on the provider record storage node. What do you think of rephrasing this?